src/components/play-timeline/play-timeline-styles.scss

```diff
@@ -0,0 +1,72 @@
+@import '../../styles/settings.scss';
+
+$timeline-width: 370px;
+$extreme-ticks-correction: 10px;
+
+.timelineContainer {
+  display: flex;
+  align-items: center;
+  background-color: transparent;
+
+  .timeline {
+    position: relative;
+    height: 8px;
+    width: $timeline-width;
+    background-color: $theme-color;
+    border-radius: 5px;
+    border-left: $extreme-ticks-correction solid $theme-color;
+    border-right: $extreme-ticks-correction solid $theme-color;
+    font-family: $font-family-1;
+    font-size: $font-size-sm;
+    font-weight: $font-weight-light;
+    color: $midnight-blue;
+
+    .timelineMarker {
+      position: absolute;
+      display: inline-block;
+      height: 20px;
+      bottom: -6px;
+      width: 4px;
+      border-radius: 2px;
+      background-color: $theme-secondary;
```

Comment:
Maybe some `transition` would be nice to smooth the movement of the marker when the years change :)